packages/next/export/index.js

```diff
@@ -69,7 +69,8 @@ export default async function (dir, options, configuration) {
     log('  copying "static build" directory')
     await cp(
       join(distDir, CLIENT_STATIC_FILES_PATH),
-      join(outDir, '_next', CLIENT_STATIC_FILES_PATH)
+      join(outDir, '_next', CLIENT_STATIC_FILES_PATH),
+      { filter: ['!.hot-update.json'] }
```

Comment:
This is fine temporarily, but there's more dev artifacts that will still be copied by this solution, for example `.next/static/development/pages`